hxntools/anc350.py

```diff
@@ -92,7 +92,7 @@ def _wait_tries(signal, value, tries=20, period=0.1):
 
 def _dc_toggle(axis, enable, freq, dc_period, off_time):
     print('Axis {} {}: '.format(axis.axis_num, axis.desc.value), end='')
-    set_and_wait(axis.frequency, freq)
+    yield from bps.mov(axis.frequency, freq)
```

Comment:
Should it be `bps.mv`?

Reply:
yes, but we might provide both

Reply:
Indeed. We will maintain both forever because no one can agree what which name is better.